main.nf

```diff
@@ -412,6 +412,9 @@ process adapter_removal {
     }
 }
 
+ch_read_files_clip.close()
+ch_clipped_reads_complexity_filtered.close()
+
```

Comment:
This looks a terrible hack, `close` shouldn't be necessary.

Reply:
Ok, I'm not yet sure which ones I actually have to close manually? Never had that issue before...